### ISSUE TITLE
Help with music and dashboard command

### ISSUE THREAD

**Issue by linuxexp** [saiteja-madha/discord-js-bot#556]
Hello sir @saiteja-madha along with other developers here,
i was interested to build discord bot of my own but didn't have that much knowledge about coding for that i ended up here on github, but not feeling regret as i saw some interesting repo of some brilliant devs including u @saiteja-madha sir. Your this repo: "https://github.com/saiteja-madha/discord-js-bot.git" helped me to finally build a multipurpose discord bot. i had seen and used other repo for bot also but i think this one's a better version in my opinion..But i recently facing problem while setting up the music and dashboard command for the bot using this repo..today im asking for help from you brilliant minds out there. ~Thank you :)

**Comment by ghost** [comment]
> Hello sir @saiteja-madha along with other developers here, i was interested to build discord bot of my own but didn't have that much knowledge about coding for that i ended up here on github, but not feeling regret as i saw some interesting repo of some brilliant devs including u @saiteja-madha sir. Your this repo: "https://github.com/saiteja-madha/discord-js-bot.git" helped me to finally build a multipurpose discord bot. i had seen and used other repo for bot also but i think this one's a better version in my opinion..But i recently facing problem while setting up the music and dashboard command for the bot using this repo..today im asking for help from you brilliant minds out there. ~Thank you :)

What is the error you even facing? and did you add CLIENT_SECRET and SESSION_PASSWORD in env before enabling dashboard in config.js? and did you add lavalink details for music system to work?

Join their support server for easy help https://discord.gg/tcXAsEGxsv

**Comment by linuxexp** [comment]
yeah i did add client_secret and i didnt know what to do with session_password so i just wrote a pass on my own..
for music command i downloaded the lavalink.jar file and created application.yml on same directory..i started the lavalink server like this: java -jar lavalink.jar on my pc terminal..i used port 8080 and replaced "localhost" with my local ip..
but the problem is i dont know what i really need to do.

**Comment by saiteja-madha** [comment]
Feel free to join our support Discord server at https://discord.com/invite/2gPy75zgbW. 
Once you’re in, just @mention me and I’ll be more than happy to assist you!
